lib/index.js

```diff
@@ -6,12 +6,31 @@ const crypto = require('crypto');
 const mustache = require('mustache');
 const path = require('path');
 const sass = require('node-sass');
+const argv = require('argv-parse');
 const verify = require('./verify');
 
 const thenify_all = require('thenify-all');
 const fs = thenify_all(require('fs-extra'), {}, ['readFile', 'writeFile', 'readdir', 'copy']);
 const tar = require('tar');
 
+const ARGV_CONFIGFILE_OPTIONS = {
+    'config-file': {
+        type: 'string'
+    }
+};
+
+const ARGV_OPTIONS = {
+    'no-tgz': {
+        type: 'boolean'
+    },
+    'path': {
+        type: 'string'
+    },
+    'urls': {
```

Comment:
Maybe `reingest-urls`? Not a blocker though.

Reply:
"reingest" is a good verb for SOMA, but the purpose of 'urls' can be any, like ingesting a specific URL for debugging.